TACC/Core-CMS
# GH-149: Polish base.html markup

## Overview

Improve the markup of `base.html`.

<details>

These changes have been pending since 2019, and sitting in a branch (since deleted in favor of this branch) for 2020+, but have been too cumbersome for which to create and test a PR (and too minor a change) until now.

</details>

## Related

- GH-149
- requires https://github.com/TACC/Core-CMS-Resources/pull/26
- required by https://github.com/TACC/Core-CMS-Resources/pull/133
- required by https://github.com/TACC/Core-Styles/pull/7

## Changes

1. Remove unnecessary ID attribute.
2. Update markup to be semantic.
3. Remove unnecessary `<div>` wrap.
4. Remove two superfluous comments.
5. Load Core-CMS-Resources and Core-Styles cleanup that depend on this.

## Screenshots

| Localhost (Markup) | Texascale (Layout)[^1] |
| - | - |
| ![GH-149](https://user-images.githubusercontent.com/62723358/164996100-bedd28d3-0e89-4cab-9c9d-0b14df75d6de.png) | ![GH-149 Texascale](https://user-images.githubusercontent.com/62723358/164996260-4eb7aae7-0d63-4af7-87fe-e99c22f7492f.png) |

[^1]: Manually manipulated markup to test.

## Testing

1. Check that markup is correctly nested.
2. Check that pulling footer out of `#cms-content` has no effect on layout.
